src/mono/mono/mini/interp/interp.c

```diff
@@ -157,7 +162,7 @@ frame_data_allocator_add_frag (FrameDataAllocator *stack, int size)
 	return new_frag;
 }
 
-static gpointer
+gpointer
```

Comment:
It might be better to add a mono_ wrapper around these, because these are perf critical, and
making them non-static might inhibit some compiler optimizations.

Reply:
is there a way to make sure they get inlined into the wrapper? the problem I had before is that there's no inlining happening for the wrappers, so the overhead was very significant

Reply:
There is MONO_ALWAYS_INLINE.

Reply:
That applies to the function though, not the call target. And I don't want to force things like stackval_to_data to always be inlined, just into the wrapper.

Reply:
Looking around, all our uses of MONO_ALWAYS_INLINE are for static functions, often in headers. Given that these wrappers touch static functions inside interp.c, will MONO_ALWAYS_INLINE work? Does it require LTO?